deployment/hasura/metadata/actions.graphql

```diff
@@ -91,6 +91,12 @@ type Query {
   ): MerlinSimulationResponse
 }
 
+type Query {
+  resourceSamples(
+    planId: Int!
+  ): ResourceSamplesResponse
```

Comment:
Would GraphQL allow us to just return a `ResourceSamples!` here?

Reply:
Unfortunately not :/ I tried that at first but you must return a `type` type, not a `scalar`. Kind of annoying because I consider both to be "types".